controller/user/manager.py

```diff
@@ -74,3 +79,16 @@ def remove_organization_from_user(user_mail: str) -> None:
         raise Exception("User has no organization")
 
     user.remove_organization(user_id, with_commit=True)
+
+
+def get_active_users(minutes: str, order_by_interaction: bool) -> User:
+    now = datetime.now()
+    last_interaction_range = now - timedelta(minutes=minutes) if minutes else None
```

Comment:
not 100% what the result should be.
if minutes aren't set is last_interaction_range = None or Now?

For readability reasons, I would either use brackets or not the inline syntax

Reply:
Done